<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dooble_clear_items</class>
 <widget class="QDialog" name="dooble_clear_items">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dooble: Clear Items</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons/icons.qrc">
    <normaloff>:/Logo/dooble.png</normaloff>:/Logo/dooble.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame {background-color: white;}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="accepted_blocked_domains">
        <property name="text">
         <string>Accepted / Blocked Domains</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bookmarks">
        <property name="text">
         <string>Bookmarks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="certificate_error_exceptions">
        <property name="text">
         <string>Certificate Error Exceptions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cookies">
        <property name="text">
         <string>Cookies</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="download_history">
        <property name="text">
         <string>Download History</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="favicons">
        <property name="text">
         <string>Favicons</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="history">
        <property name="text">
         <string>History</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="visited_links">
        <property name="text">
         <string>Visited Links</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>accepted_blocked_domains</tabstop>
  <tabstop>bookmarks</tabstop>
  <tabstop>certificate_error_exceptions</tabstop>
  <tabstop>cookies</tabstop>
  <tabstop>download_history</tabstop>
  <tabstop>favicons</tabstop>
  <tabstop>history</tabstop>
  <tabstop>visited_links</tabstop>
 </tabstops>
 <resources>
  <include location="../Icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dooble_clear_items</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dooble_clear_items</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
